docs/_static/custom.css

```diff
@@ -1,14 +1,30 @@
-div.body table.widget-descriptions td p
-{
-    text-align: center;
+body {
+    --color-beta: #c8c8c8;
+    --color-beta-dark: #484848;
+}
+body[data-theme="dark"] {
+    --color-beta: var(--color-beta-dark);
 }
-div.body table.widget-support th p,
-div.body table.widget-support td p {
+@media (prefers-color-scheme: dark) {
+    body:not([data-theme="light"]) {
```

Comment:
Furo lets you select dark/light/default mode using the sun/moon icon at the top right.